src/main/java/frc/config/Config.java

```diff
@@ -41,24 +44,30 @@ public static double getDouble(String key) {
 
 	public static void start() {
 		map = new HashMap<>();
+		try {
+			Scanner nameScanner = new Scanner(new File("/home/lvuser/name.192"));
+			String name = nameScanner.nextLine();
+			nameScanner.close();
+			String fileName = name + ".txt";
+			System.out.println("reading from file " + fileName);
+			File f = new File(Filesystem.getDeployDirectory(), fileName);
+			Scanner scanner = new Scanner(f);
+			while (scanner.hasNextLine()) {
+				String line = scanner.nextLine();
 
-		String fileName = "hauntedhouse2019.txt";
-		System.out.println("reading from file " + fileName);
-		InputStream stream = Config.class.getResourceAsStream(fileName);
-		Scanner scanner = new Scanner(stream);
-		while (scanner.hasNextLine()) {
-			String line = scanner.nextLine();
-
-			while (line.startsWith(" "))
-				line = line.substring(1);
+				while (line.startsWith(" "))
+					line = line.substring(1);
 
-			if (line.length() > 0 && line.charAt(0) != '#') {
-				String[] splitted = line.trim().split("=");
-				if (splitted.length == 2)
-					map.put(splitted[0], splitted[1]);
+				if (line.length() > 0 && line.charAt(0) != '#') {
+					String[] splitted = line.trim().split("=");
+					if (splitted.length == 2)
+						map.put(splitted[0], splitted[1]);
+				}
 			}
+			scanner.close();
```

Comment:
This whole try block is a nice candidate for turning into a function. Especially since we've created a `Scanner` class, we could consider making some sort of `processLine()` function.